src/components/POLine/Item/ContributorForm.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  Field,
+  FieldArray,
+} from 'redux-form';
+import {
+  Button,
+  Col,
+  Row,
+  TextField,
+} from '@folio/stripes/components';
+
+class ContributorForm extends Component {
+  constructor(props) {
+    super(props);
+    this.addFields = this.addFields.bind(this);
+    this.removeFields = this.removeFields.bind(this);
+    this.renderForm = this.renderForm.bind(this);
+    this.renderSubForm = this.renderSubForm.bind(this);
+  }
+
+  addFields(fields) {
+    fields.push('');
+  }
+
+  removeFields(fields, index) {
+    fields.remove(index);
+  }
+
+  renderForm({ fields }) {
+    return (
+      <Row>
+        <Col xs={12}>
+          {fields.length === 0 &&
+            <Col xs={12}>
+              <div>
+                <em>
+                  <FormattedMessage id="ui-orders.itemDetails.addContributor" />
+                </em>
+              </div>
+            </Col>
+          }
+          {fields.map(this.renderSubForm)}
+        </Col>
+        <Col xs={12} style={{ paddingTop: '10px' }}>
+          <Button onClick={() => this.addFields(fields)}>
+            <FormattedMessage id="ui-orders.itemDetails.addContributorBtn" />
+          </Button>
+        </Col>
+      </Row>
+    );
+  }
+
+  renderSubForm(elem, index, fields) {
+    return (
+      <Row key={index}>
+        <Col xs={10}>
+          <Field
+            label={<FormattedMessage id="ui-orders.itemDetails.contributor" />}
+            name={`${elem}.contributor`}
+            id={`${elem}.contributor`}
+            component={TextField}
+            fullWidth
+          />
+        </Col>
+        <Col xs={2} style={{ paddingTop: '4px' }}>
+          <br />
```

Comment:
it's not semantic to use tag `<br />` for spacing purposes. It's purpose to make new line in html code, spacing should be done via css - margins, paddings, alignments or display attributes